securedrop/journalist.py

```diff
@@ -298,6 +297,13 @@ def admin_edit_user(user_id):
     if request.method == 'POST':
         if request.form['username']:
             new_username = request.form['username']
+
+            try:
+                Journalist.check_username_acceptable(new_username)
+            except InvalidUsernameException as e:
+                flash('Invalid username: ' + str(e))
```

Comment:
Should be: 

`flash('Invalid username: ' + str(e), 'error')`

Otherwise the flashed message style is inconsistent: 

![screen shot 2017-08-24 at 11 10 23 am](https://user-images.githubusercontent.com/7832803/29681563-eb6b20dc-88bc-11e7-9ce3-aea6e3e5d57e.png)